org.code.toboggan.network/META-INF/MANIFEST.MF

```diff
@@ -12,3 +12,4 @@ Require-Bundle: org.eclipse.core.runtime,
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Bundle-ActivationPolicy: lazy
 Export-Package: org.code.toboggan.network
+Import-Package: com.google.common.collect
```

Comment:
yes, but there are two ways to include packages in a plugin. The first is to include a bundle, which we have been doing consistently. Including a bundle automatically includes all the packages inside it. Including a package doesn't specify where it comes from (so this dependency could be overwritten at runtime by another plugin providing it) and we don't do this at all. This is what you're doing here.